src/graph/GraphViewServer.ts

```diff
@@ -135,50 +136,51 @@ export class GraphViewServer extends EventEmitter {
     const start = Date.now();
 
     try {
-      await this._actionHandler.callWithTelemetry("cosmosDB.gremlinQuery", async (telemetryProperties, measurements) => {
-        this._pageState = {
+      const graphViewServier: GraphViewServer = this;
+      await AzureActionHandler.callWithTelemetry("cosmosDB.gremlinQuery", reporter, async function (this: IActionContext): Promise<void> {
```

Comment:
I'm liking the "this" thing less now that I see it. :-(  But not worth rehashing the other PR for.

Might consider immediately doing:
  let actionContext = this;

Reply:
Yeah I'm open to changing it, but I agree it's not worth re-hashing right now